cloud/pkg/devicecontroller/config/buffer.go

```diff
@@ -9,11 +9,32 @@ import (
 // UpdateDeviceStatusBuffer is the size of channel which save update device status message from edge
 var UpdateDeviceStatusBuffer int
 
+// DeviceEventBuffer is the size of channel which save device event from k8s
+var DeviceEventBuffer int
+
+// DeviceModelEventBuffer is the size of channel which save devicemodel event from k8s
+var DeviceModelEventBuffer int
+
 func init() {
 	if psb, err := config.CONFIG.GetValue("devicecontroller.buffer.update-device-status").ToInt(); err != nil {
 		UpdateDeviceStatusBuffer = constants.DefaultUpdateDeviceStatusBuffer
 	} else {
 		UpdateDeviceStatusBuffer = psb
 	}
-	log.LOGGER.Infof("Update device status buffer: %d", UpdateDeviceStatusBuffer)
+	log.LOGGER.Infof("devicecontroller Update device status buffer: %d", UpdateDeviceStatusBuffer)
+
+	if deb, err := config.CONFIG.GetValue("devicecontroller.device-event-buffer").ToInt(); err != nil {
```

Comment:
As @sids-b suggests, it is better to add configurations under buffer like the line 19 mentions
"devicecontroller.buffer.update-device-status"
So the folloing modification is welcome:
Replace "devicecontroller.device-event-buffer" with "devicecontroller.buffer.device-event"
Replace "devicecontroller.device-model-event-buffer" with "devicecontroller.buffer.device-model-event"

Reply:
+1